javascript/node/selenium-webdriver/lib/webdriver.js

```diff
@@ -1229,6 +1229,14 @@ class WebDriver {
     if (target && cdpTargets.indexOf(target.toLowerCase()) === -1) {
       throw new error.InvalidArgumentError('invalid target value')
     }
+
+    if (debuggerAddress.match(/\/se\/cdp/)) {
```

Comment:
if we are returning the `ws` here when passing in `se:cdp` we can just return it straight or do we have to make a request to get the `ws` address?

Reply:
We do not need to make a request to get the `ws` address. When the request is made here, it asks the Grid to upgrade to a socket and then the Grid makes the request to `/json/version` to get the address and uses that.

Reply:
ok! 👍